Move to immutable goals: update proj kickoff

tannerwelsh opened the pull request:
- No more learning contract creation/review process as default (only in
  cases where specs are unclear)
- Assume that goals have pre-defened goal repo w/ CONTRACT.md

See https://github.com/GuildCrafts/web-development-js/pull/155 for more context on the switch to using goals as repos.

tannerwelsh committed:
Move to immutable goals: update proj kickoff

- No more learning contract creation/review process as default (only in
  cases where specs are unclear)
- Assume that goals have pre-defened goal repo w/ CONTRACT.md

shereefb commented on Learning_Guide/Kickoff.md:
```diff
@@ -6,106 +6,41 @@ The purpose of a project is for the team to achieve the goal through practicing
-- Create a repo for your project, and make a simple first commit. Your first commit should contain a README.md file with your project's name, team members, a link to the goal and a LICENSE.MD file.
+In most cases, the artifact will be a GitHub repository in the account of a learner on the team that is forked from the goal repository.
+
+- Create a repo for your project by forking the goal repository\* and make a simple first commit. Your first commit should write or update the README.md file with your project's name, team members, a link to the goal and a LICENSE file (unless it already exists).
```
If it has a repo. 
Otherwise create your own repo and add the contract.md file

shereefb commented on Learning_Guide/Kickoff.md:
```diff
@@ -6,106 +6,41 @@ The purpose of a project is for the team to achieve the goal through practicing
 1. It guides and focuses your efforts for the rest of the cycle.
 2. It defines completeness and quality for your code reviewers during the reflection phase.
-3. It is a means for you to practice **project estimation**, **specing**, and **goal setting**. These are critical skill for any software developer to develop (and really difficult ones)
+3. It is a means for you to practice **project estimation**, **spec'ing**, and **goal setting**. These are critical skill for any software developer to develop (and really difficult ones).
```
Hmmm... are they still practicing these things?

tannerwelsh replied:
Not really. Will remove.

shereefb commented on Learning_Guide/Kickoff.md:
```diff
@@ -6,106 +6,41 @@ The purpose of a project is for the team to achieve the goal through practicing
-To set quality criteria, choose 2 or 3 rules or guidelines that, as a team, you agree to follow to ensure a quality artifact. Add these to the Learning Contract.
-
-For example, a quality criteria might be "Follow object-oriented principles in all program design" or "Write good commit messages using these rules: http://chris.beams.io/posts/git-commit/".
+Review the Learning Contract included in the goal. You may need to improve it if the specs are unclear or the quality rubric is incomplete. If so, make changes in a pull request and then submit your PR to [prrr](http://prrr.apps.learnersguild.org/).
```
Hmmm... a little tricky here. Maybe make it clear somehow that this is an exception. 
Also, they should submit the changes to the goal librarian as well

tannerwelsh replied:
Yeah. I agree. Removing it. Instead just suggesting that people update the source contract via a PR.

shereefb commented:
lgtm

tannerwelsh committed:
Clarify learning contract creation

- Incorporate @shereefb's feedback
- Reference contracts/ dir of web-dev-js repo
- Don't assume goals have goal repos

tannerwelsh commented:
Incorporated some of the feedback. Moving to immutable contracts coming from the goal library. Holding until https://github.com/GuildCrafts/web-development-js/pull/160 is merged.
